engineering/ember.md

```diff
@@ -25,6 +26,17 @@ fullName: Ember.computed('user.firstName', 'user.lastName', {
 })
 ```
 
+## Templates
+
+### Don't use partials
+
+Use components instead of partials.
```

Comment:
I tried to write one but it's hard to create one that accurately conveys the "bad" without making it really big. Open to adding one if we can come up with a good one though